projects/ssr-tests/src/utils/proxy.utils.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as http from 'http';
+import httpProxy, { ProxyResCallback } from 'http-proxy';
+
+const proxy = httpProxy.createProxyServer({
+  secure: false,
+  // selfHandleResponse: true,
+});
```

Comment:
[MEDIUM]
To avoid being prone to interferences between tests, I suggest to move it to the `startBackendProxyServer`.
Otherwise, multiple calls of this function operate on the same global shared object `proxy`

Reply:
Good point